packages/ui/src/accounts/modals/MoveFoundsModal/MoveFoundsTransferableModal.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+
+import { AccountInfo } from '@/accounts/components/AccountInfo'
+import { AddressToBalanceMap, Account } from '@/accounts/types'
+import { ButtonPrimary } from '@/common/components/buttons'
+import { HelpIcon } from '@/common/components/icons'
+import { Modal, ModalHeader, BalanceInfoInRow, InfoTitle, InfoValue, ModalFooter } from '@/common/components/Modal'
+import { TextMedium, TokenValue } from '@/common/components/typography'
+import { Colors } from '@/common/constants'
+import { spacing } from '@/common/utils/styles'
+
+import { ModalBody, MemberRow } from './styles'
+
+interface Props {
+  onClose: () => void
+  onManageAccountsClick: () => void
+  price: number
+  balances: AddressToBalanceMap
+  allAccounts: Account[]
```

Comment:
I think that this modal should obtain as a prop set of the accounts to render.

This modal has only an informational character, the actual check might be invoked earlier in other steps of the application. That's because we need to inform the user that there are transferable funds that can be used even though none of the accounts have the required stake  ([the move funds case](https://github.com/Joystream/pioneer/issues/305#issuecomment-845803741)).

In other words, we might have actions:

*   [apply for a role](https://joystream.gitbook.io/joystream-handbook/governance/working-groups#staking)
*   [vote](https://joystream.gitbook.io/joystream-handbook/governance/council#vote) in elections
*   [create a proposal](https://joystream.gitbook.io/joystream-handbook/governance/proposals#staking)
*   apply as a candidate

and all of them will require some `stake` to be locked when issuing an action. The App will check the required stake and if it finds out that case 1 or 2 is met then it will display the `MoveFundsTransferableModal` or `MoveFundsLockedModal`.